.gitignore

```diff
@@ -1,9 +1,4 @@
-node_modules
-bower_components
-dist/assets/index.html
-dist/assets/main.css
-dist/assets/main.css.map
-dist/assets/main.js
-dist/assets/main.js.map
-dist/translations/
-dist/tmp/
+tmp/
+dist/
+node_modules/
+coverage/
```

Comment:
can we add `npm-debug.log` to this. it always comes up and bugs me :P

Reply:
👍